lib/index.js

```diff
@@ -31,6 +45,57 @@ internals.implementation = function (server, options) {
                 return reply(Boom.unauthorized(null, 'Access Token'));
             }
     
+            var parts = parseCookies(authorization);
```

Comment:
@timotgl I think I understand this part correctly but I'm wondering how to retrieve cookieName from the routes that need to state/unstate it. Maybe there is a "hapi standard" way to add the plugin options to the request object so that I could use in my route handler something like:
```javascript
reply().state(request.auth.cookieId).code(200);
```
I'll try something